samples/custom_dispatch/cpu/mlp_plugin/CMakeLists.txt

```diff
@@ -108,4 +113,65 @@ iree_lit_test_suite(
     "hostonly"
 )
 
+# Append the includes used for this file to the pdll_compilation_commands
+file(APPEND ${CMAKE_BINARY_DIR}/pdll_compile_commands.yml
+  "--- !FileInfo:\n"
+  "  filepath: \"${CMAKE_CURRENT_SOURCE_DIR}/mlp_all_specs.pdll\"\n"
+  "  includes: \"${TORCH_MLIR_ROOT_DIR}/include;${CMAKE_CURRENT_SOURCE_DIR};${MLIR_INCLUDE_DIRS}\"\n"
+)
+
+iree_lit_test_suite(
+  NAME
+    mlp_pdll_linalg_example
+  SRCS
+    "mlp_linalg_pdll.mlir"
+  TOOLS
+    FileCheck
+    iree-compile
+    iree-run-module
+    mlir-pdll
+    iree_samples_custom_dispatch_cpu_mlp_plugin
+  DATA
+    "mlp_all_specs.pdll"
+  LABELS
+    "driver=local-sync"
+    "hostonly"
+)
+
+iree_lit_test_suite(
+  NAME
+    mlp_pdll_torch_example
+  SRCS
+    "mlp_torch_pdll.mlir"
```

Comment:
Tests using input dialects (torch, tosa, stablehlo) should be only defined if their matching CMake option is set. Otherwise, developers who disable an input dialect will get unactionable test failures.

Here are the available options: https://github.com/openxla/iree/blob/a2476ceef53f6d4ec0f7a9cf36892647626d324b/CMakeLists.txt#L418-L420

Here is an example of how to use one:
https://github.com/openxla/iree/blob/a2476ceef53f6d4ec0f7a9cf36892647626d324b/compiler/bindings/python/test/tools/CMakeLists.txt#L18-L25